code/lib/manager-api/src/lib/stories.ts

```diff
@@ -196,9 +196,7 @@ export const transformStoryIndexToStoriesHash = (
       if (parent === id) {
         throw new Error(
           dedent`
-          Invalid part '${name}', leading to id === parentId ('${id}'), inside title '${title}'
-          
-          Did you create a path that uses the separator char accidentally, such as 'Vue <docs/>' where '/' is a separator char? See https://github.com/storybookjs/storybook/issues/6128
```

Comment:
I like the solution proposed here:
https://github.com/storybookjs/storybook/pull/26911/files

WDYT? @hmcostaa

Reply:
That was one possible solution. I just thought of an additional feature that would complement this problem by rendering the correctly typed stories.